hack/verify-spelling.sh

```diff
@@ -17,8 +17,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+DIR=$(dirname $0) 
+
 # Install tools we need
-go install ../../github.com/client9/misspell/cmd/misspell
+go install ${DIR}/../../../github.com/client9/misspell/cmd/misspell
```

Comment:
Just curious, why `../..` is used here? I think it's same with `go install github.com/client9/misspell/cmd/misspell`